src/dataDisplay/icon/icon.jsx

```diff
@@ -90,6 +91,10 @@ class Icon extends React.PureComponent {
         }
     }
 
+    onMouseDown(event) {
+        event.preventDefault();
+    }
```

Comment:
Is this _always_ the right thing to do?  There are no exceptions, where this is not correct?